weblab/conftest.py

```diff
@@ -43,9 +42,9 @@ def add_version(entity,
         return commit
 
     @staticmethod
-    def add_fake_version(entity, visibility, date=None):
+    def add_fake_version(entity, visibility='private', date=None):
```

Comment:
this is just a question for my own understanding pls... why are we changing this to be explicitly private here.

Reply:
This just sets a default visibility as private if none is supplied by the caller. I changed it to be consistent with `add_version`, so when moving tests to use 'fake' versions (i.e. purely in the DB, no git repository) I didn't need to add explicit visibilities when they were previously using the default.

Reply:
thanks make sense. The change doesn't appear to be in the latest code for this branch, so I'm confused why its showing up in the diff

Reply:
It's in the last commit 224e49d I think, which is after I requested the review. So your local copy might be outdated?

Reply:
yes that was the case. Pycharm wasn't doing what I expected. all good now. thanks